ocs_ci/ocs/ui/base_ui.py

```diff
@@ -55,6 +56,54 @@
 logger = logging.getLogger(__name__)
 
 
+def wait_for_element_to_be_clickable(locator, timeout=30):
+    """
+    Wait for an element to be clickable.
+
+    Args:
+        locator (tuple): A tuple containing the locator strategy (e.g., By.ID, By.XPATH) and the locator value.
+        timeout (int): Maximum time (in seconds) to wait for the element to be clickable. Defaults to 30 seconds.
+
+    Returns:
+        selenium.webdriver.remote.webelement.WebElement: The clickable web element.
+
+    """
+    wait = WebDriverWait(SeleniumDriver(), timeout)
+    try:
+        web_element = wait.until(
+            ec.visibility_of_element_located((locator[1], locator[0]))
+        )
+    except TimeoutException:
+        take_screenshot()
+        copy_dom()
+        raise
+    return web_element
```

Comment:
I don't think we need this function as ec already has `element_to_be_clickable` & `visibility_of_element_located` by default which should be used directly.

Reply:
disagree. ec is an expected_conditions function, it should not be called from the test because it makes a number of lines and requires locator, also there is logic here to take a screenshot and copy dom, which is an only source to identify failure on UI during RP analisys.